pkg/webhook/webhook.go

```diff
@@ -55,15 +59,49 @@ func (q *QuayIntegrationMutator) Handle(ctx context.Context, req admission.Reque
 			}
 		}
 	} else {
-
-		admissionResponse = getAdmissionResponseForBuild(build, &quayIntegration)
+		// Check if builder service account has secret
+		hasSecret, serviceAcctErr := q.checkSecretForBuilderServiceAccount(ctx, &req, &quayIntegration)
```

Comment:
@sabre1041 , I discussed this with the team and the consensus was that the [build controller does the retry](https://github.com/openshift/openshift-controller-manager/blob/4b96187495df95b35b3728d4966ca0720b57541c/pkg/build/controller/build/build_controller.go#L1779) when the webhook handler blocks the creation of a `builds` resource so the retry won't be necessary here. Lmk if you feel otherwise